Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -603,5 +603,6 @@ It can efficiently compress both junk data and encrypted Wii data.
     <string name="disabled_gc_overlay_notice">GameCube Controller 1 is set to \"None\"</string>
     <string name="ignore_warning_alert_messages">Ignore for this session</string>
     <string name="make_sure_continuous_scan_enabled">Please make sure Continuous Scanning is enabled in Core Settings.</string>
+    <string name="string">string</string>
```

Comment:
This probably shouldn't be here.

Reply:
oh crap, i think the android studio design tool did that